force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -6264,6 +6264,14 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Records Processed</shortDescription>
         <value>Records Processed</value>
     </labels>
+    <labels>
```

Comment:
@ErinWiedemer @sam-knox can you review the custom labels for the UI.  [UX design](https://www.figma.com/file/bEsmvar7D5lSBS0ZuR5fz8/Full-Refund-from-Payment-Record?node-id=0%3A1)

Reply:
I'll take the initial review of these labels. @ErinWiedemer, once you're back from PTO let's plan to take these labels to one of our internal UI reviews. We'll handle any additional edits in a future PR.

Reply:
Sounds good @sam-knox. Feel free to put time on my calendar for it.

Reply:
Is there a new process to add the doc team to review the PR or adding you all on the PR  as reviewers are ok?

Reply:
Adding us as reviewers on the PR is just fine, @andrewyu-salesforce. Thanks!